packages/canon/docs/components/Text/styles.css

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-.text {
+.sb-text {
```

Comment:
we're gonna need the inverse solution to this, as in we can't require these plain class names to be unused

Reply:
Yes. I'm planning to do this for all components in a separate PR.